extract/ocr_db_data.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+sys.path.append("../")
+from wand.image import Image
+from PIL import Image as PI
+import io
+import os
+import pyocr
+import pyocr.builders
+import pandas as pd
+from lib.path import data_file_path
+
+
+pyocr.tesseract.TESSERACT_CMD = r"C:/Program Files/Tesseract-OCR/tesseract.exe"
+
+
+dir_path = os.chdir(data_file_path("raw/new_orleans_csc/appeal_hearing_pdfs"))
+for file in os.listdir(dir_path):
+    if file.endswith(".pdf"):
+        tools = pyocr.get_available_tools()
+        if len(tools) == 0:
+            print("No OCR tool found!")
+            sys.exit(1)
```

Comment:
Rather than print and then exit, just raise an error instead. Just use `Exception` in this case.

Reply:
Smart. Thanks.